src/pages/settings/InitialSettingsPage.tsx

```diff
@@ -182,39 +175,38 @@ function InitialSettingsPage(props) {
         };
 
         if (NativeModules.HybridAppModule) {
-            const hybridAppMenuItems = _.filter(
-                [
-                    {
-                        translationKey: 'initialSettingsPage.returnToClassic',
-                        icon: Expensicons.RotateLeft,
-                        shouldShowRightIcon: true,
-                        iconRight: Expensicons.NewWindow,
-                        action: () => NativeModules.HybridAppModule.closeReactNativeApp(),
+            const hybridAppMenuItems: MenuData[] = [
+                {
+                    translationKey: 'initialSettingsPage.returnToClassic' as const,
+                    icon: Expensicons.RotateLeft,
+                    shouldShowRightIcon: true,
+                    iconRight: Expensicons.NewWindow,
+                    action: () => {
+                        NativeModules.HybridAppModule.closeReactNativeApp();
                     },
-                    ...defaultMenu.items,
-                ],
-                (item) => item.translationKey !== 'initialSettingsPage.signOut' && item.translationKey !== 'initialSettingsPage.goToExpensifyClassic',
-            );
+                },
+                ...defaultMenu.items,
+            ].filter((item) => item.translationKey !== 'initialSettingsPage.signOut' && item.translationKey !== 'exitSurvey.goToExpensifyClassic');
```

Comment:
@allroundexperts  Changing this key since this looked like a mistake because it was always false